sampleconfig/orderer.yaml

```diff
@@ -50,6 +50,12 @@ General:
         # ServerTimeout is the duration the server waits for a response from
         # a client before closing the connection.
         ServerTimeout: 20s
+
+    # Max message size in bytes the GRPC server and client can receive
+    MaxRecvMsgSize: 104857600
+    # Max message size in bytes the GRPC server and client can send
```

Comment:
```suggestion
    # Max message size in bytes the GRPC server and client can send.
    # Since all nodes should be consistent it is recommended to keep the default value of 100MB.
```